src/orqviz/elastic_band/data_structures.py

```diff
@@ -1,11 +1,12 @@
 from __future__ import annotations
 
-from typing import Callable, NamedTuple
+from typing import Callable, NamedTuple, Tuple
 
 import numpy as np
 from scipy.interpolate import interp1d
 
-from ..aliases import ArrayOfParameterVectors, Weights
+from ..aliases import ArrayOfParameterVectors, LossFunction, ParameterVector, Weights
+from ..geometric import _norm_of_arrayofparametervectors
```

Comment:
Since you are using `_norm_of_arrayofparametervectors` in a different module than it comes from (i.e. in `elastic_band` and not in `geometric`, I think it makes more sense to not make it private?
Not 100% sure though, thoughts @alexjuda?

Reply:
That's a good point. It is only a helper function for us though. I doubt that anyone would need that apart from us.

Reply:
I can recommend a 3-fold distinction for project symbols – public/internal/fileprivate based on a convention with leading underscores in the module and symbol names;
- public: `module`, `function`
- internal: `_module`, `function`
- fileprivate: w/e, `_function`
The above use case looks like an "internal" one :). Thoughts?

Reply:
@alexjuda  In our case the content of `geometric` packages should be public, so not really ;) 
@MSRudolph fine 🤷‍♂️